pinot-core/src/main/java/org/apache/pinot/core/realtime/converter/stats/MutableColumnStatistics.java

```diff
@@ -33,15 +33,15 @@
  *
  * TODO: Gather more info on the fly to avoid scanning the segment
  */
-public class RealtimeColumnStatistics implements ColumnStatistics {
+public class MutableColumnStatistics implements ColumnStatistics {
```

Comment:
Do we ever read this from the controller? Can you confirm that this is only used by the server?

If this is used by multiple components, renaming may cause incompatibility during the deployement.

Reply:
The implementation of `ColumnStatistics` is used during segment creation. As long as there is no protocol change, I don't think renaming will cause incompatibility during the deployment.